config/config.py

```diff
@@ -70,3 +70,9 @@ def target_dir(self) -> str:
             return os.getcwd()
         return target_path
 
+    @property
```

Comment:
A duplication of lines 66-71, can be removed 🙂

Reply:
removed